.github/workflows/publish_release.yml

```
@@ -70,6 +73,19 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
 
+      - name: Extract release notes from CHANGELOG
+        if: "!contains(github.ref, '-rc.')"
```

You only want to get the changelog if it's a full release, right? In that case, you can do

```suggestion
        if [ "$VERSION" == "$LATEST_TAG" ]
```

---

Yes. I thought I did test this condition & it didn't work, but there doesn't seem to be a commit for it in my history. I might have only done it locally. Can try running another RC with this just to be sure.